tests/pkg_winc1500_netdev/main.c

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (C) 2017 Bumsik Kim <k.bumsik@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the WINC1500 driver
+ *
+ * @author      Bumsik Kim <k.bumsik@gmail.com>
+ *
+ * @}
+ */
+
+#include "board.h"
+#include "shell.h"
+#include <string.h>
+
+#include "winc1500.h"
+#include "winc1500_params.h"
+
+/* Shell functions */
+static int _init(int argc, char **argv);
+static int _scan(int argc, char **argv);
+static int _connect(int argc, char **argv);
+static int _disconnect(int argc, char **argv);
+static int _info(int argc, char **argv);
+
+#define MAIN_QUEUE_SIZE     (8U)
+static msg_t _main_msg_queue[MAIN_QUEUE_SIZE];
+
+#ifdef MODULE_NETDEV_ETH
+/* Import winc1500 objects from auto_init_winc1500.c */
+extern winc1500_t winc1500_devs[];
+winc1500_t *winc1500_dev = &winc1500_devs[0];
```

Comment:
This is currently the only use of global variable. This is needed after adding multiple WiFi modules support.
Because, in the application code, the user need to select a WiFi device when asking for scanning/connecting/etc, the user should select one of the device descriptors `winc1500_devs` defined in `auto_init_winc1500.c`.
@aabadie What do you think about this?